test/parallel/test-timers-promisified.js

```diff
@@ -10,6 +10,11 @@ const { promisify } = require('util');
 const setTimeout = promisify(timers.setTimeout);
 const setImmediate = promisify(timers.setImmediate);
 
+process.on('multipleResolves', (type, promise, reason) => {
+  console.error('multipleResolves', type, promise, reason);
+  throw new Error('multipleResolves');
+});
```

Comment:
```suggestion
process.on('multipleResolves', common.mustNotCall());
```

Reply:
That's what I initially wrote but if you want to debug this you would still need to rewrite it to print the info, so I'd prefer to leave it as is.

Reply:
Yeah, done https://github.com/nodejs/node/pull/33951 =)
Will update this shortly.